### PR TITLE
ENCD-3879 Pin Alembic version

### PR DESCRIPTION
**Pull request by keenangraham**

### CODE REVIEW

File: versions.cfg [hunk 1]
```diff
@@ -256,3 +256,8 @@ elasticsearch = 5.4.0
 
 # Added by buildout at 2018-01-17 17:26:37.230945
 selenium = 3.8.1
+
+# Required by:
+# encoded==66.0
+alembic = 0.9.8
+python-editor = 1.0.3
```
Comment (caseylitton) [inline versions.cfg]: What is the python-editor for?

Comment (davisagli) [reply versions.cfg]: It's a dependency of `alembic`. It looks like a small utility to help with opening the user's preferred text editor from Python.

**Comment by forresttanaka** [comment]
As far as usage, I did a clean buildout in this branch and had no modified files afterwards, so it looks good.

File: versions.cfg (hunk 1 above)
Comment (keenangraham) [reply versions.cfg]: Dependency of Alembic.

**Pull request merged**